explorer/src/lib/stores/marketDataStore.js

```diff
@@ -43,9 +43,19 @@ const marketDataStore = derived(
   initialState
 );
 
+function isDataStale() {
+  const { error, isLoading, lastUpdate } = get(marketDataStore);
+
+  return (
+    !!lastUpdate &&
+    (error !== null || (!isLoading && Date.now() > +lastUpdate + fetchInterval))
+  );
+}
+
```

Comment:
Wondering if something like this is cleaner to read:

```
function isDataStale() {
  const { error, isLoading, lastUpdate } = get(marketDataStore);

  if (!lastUpdate) {
    return false;
  }

  const isError = error !== null;
  const isExpired = Date.now() > +lastUpdate + fetchInterval;

  return isError || (!isLoading && isExpired);
} 
```

🧐

Reply:
I get your point but, don't know, I'm doubtful in a thing this short. Obviously your mileage may vary, as they say.

For me it can be actually less readable as I have to read the variables and then read how they are combined in the condition.